k-distribution/include/builtin/domains.k

```diff
@@ -863,6 +863,10 @@ endmodule
 
 module BYTES-SYMBOLIC [symbolic, kast]
   imports BYTES-IN-K
+
+  rule #Ceil(padRightBytes(_, LEN, VAL)) => {(0 <=Int LEN andBool 0 <=Int VAL andBool VAL <Int 256) #Equals true} [anywhere, simplification]
```

Comment:
I have the same comment I have on the other PR. If these are intended for the haskell backend, they should not be in the module tagged [kast].

Reply:
@dwightguth Should I create a separate module just for these rules?

Reply:
They should be in a module tagged `kore` or `kore, symbolic`